src/main/kotlin/com/github/suusan2go/kotlinfillclass/inspections/FillSameValuesInspection.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.suusan2go.kotlinfillclass.inspections
+
+import com.github.suusan2go.kotlinfillclass.helper.PutArgumentOnSeparateLineHelper
+import com.intellij.codeInspection.ui.MultipleCheckboxOptionsPanel
+import org.jetbrains.kotlin.builtins.KotlinBuiltIns
+import org.jetbrains.kotlin.descriptors.ValueParameterDescriptor
+import javax.swing.JComponent
+
+class FillSameValuesInspection : BaseFillClassInspection(withoutDefaultValues = false) {
+
+    override fun getConstructorPromptTitle(): String {
+        return "Fill class constructor with same name values"
```

Comment:
With this name, I expect it to work like `foo(bar = "bar")`, but it is actually `foo(bar = bar)`, so I think it would be better to rename it as `Fill class constructor with variables of the same name`.

Reply:
I use this feature for fill create objects on convert classes from data layer to domain layer